sources/packages/backend/libs/utilities/src/config/config.service.ts

```diff
@@ -31,6 +31,7 @@ export class ConfigService {
         institution: process.env.KEYCLOAK_CLIENT_INSTITUTION,
         aest: process.env.KEYCLOAK_CLIENT_AEST,
         supportingUsers: process.env.KEYCLOAK_CLIENT_SUPPORTING_USERS,
+        external: process.env.KEYCLOAK_CLIENT_EXTERNAL,
```

Comment:
Is this variable going to be added to the CI/CD?

Reply:
Yes, sorry. I forgot that. I'm going to add it.

Reply:
My understanding is that we will no longer use it and it was removed, hence resolving the comment 😉